py3status/modules/netdata.py

```diff
@@ -59,12 +59,29 @@ class Py3status:
     low_traffic = 400
     med_speed = 60
     med_traffic = 700
-    nic = 'eth0'
+    nic = None
 
     def __init__(self):
         self.old_transmitted = 0
         self.old_received = 0
 
+    def post_config_hook(self):
+        if self.nic is None:
+            """
+            Get default gateway directly from /proc.
+            """
```

Comment:
:-) Question. I wanted to type `self.py3.notify_user('netdata: cannot find a nic. selected lo instead.')` but it'll break the column barrier. I use `\` then next line all the way left... `instead.`)` -- Not aligned. Worked fine, but seems wrong?